app/src/main/java/fr/free/nrw/commons/contributions/Contribution.java

```diff
@@ -68,6 +68,7 @@ public Contribution createFromParcel(Parcel parcel) {
     public String decimalCoords;
     public boolean isMultiple;
     public String wikiDataEntityId;
+    public String wikiItemName;
```

Comment:
If you add a field to contribution you have to add it to the parcelable methods as well or you break the contract

Reply:
I have a doubt. why isn't wikiDataEntityId added to parcelable method

Reply:
The Parcelable interface comes with some necessities for a class to implement, you must be able to write the object to a parcel and you must be able to read from a Parcel.

See the [example](https://developer.android.com/reference/android/os/Parcelable)

Relevant lines

```
     public void writeToParcel(Parcel out, int flags) {
         out.writeInt(mData);
     }

     private MyParcelable(Parcel in) {
         mData = in.readInt();
     }
```

Reply:
> why isn't wikiDataEntityId added to parcelable method

Probably someone messed up because it is difficult to see from a small change that this class is parcelable. You would have to know it off the top of your head